steps/reasons-for-appealing/evidence-upload/EvidenceUpload.js

```diff
@@ -83,35 +120,47 @@ class EvidenceUpload extends Question {
                 uploadEv: b.documents[0].originalDocumentName,
                 link: b.documents[0]._links.self.href
               };
+
               return fs.unlink(pathToFile, next);
             }
             return next(forwardingError);
           });
         });
-      }
-      return next();
-    });
+      });
+    }
+    return next();
   }
 
   get middleware() {
-    return [EvidenceUpload.handleUpload, ...super.middleware];
+    return [...super.middleware, EvidenceUpload.handleUpload];
   }
 
   get form() {
     return form({
       uploadEv: text.joi(
-        'Please choose a file',
+        this.content.fields.uploadEv.error.required,
         Joi.string().required()
+      ).joi(
+        this.content.fields.uploadEv.error.wrongFileType,
+        Joi.string().disallow(wrongFileTypeError)
+      ).joi(
+        this.content.fields.uploadEv.error.maxFileSizeExceeded,
+        Joi.string().disallow(maxFileSizeExceededError)
       ),
-      link: text.joi('Unexpected error', Joi.string().required())
+      link: text.joi('', Joi.string().optional())
```

Comment:
what's this for?

Reply:
I mean there "link is part of the form so send it, but do not validate it, because its existence is guaranteed by the presence of `uploadEv`, and I don't want to display additional errors".

Makes sense? Since it's not mapped to a field, it wouldn't be sent if it wasn't here, isn't it?